span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/rawspan/ByPassPredicate.java

```diff
@@ -1,22 +1,36 @@
 package org.hypertrace.core.spannormalizer.rawspan;
 
 import com.typesafe.config.Config;
+import java.util.Collections;
+import java.util.List;
 import org.apache.kafka.streams.kstream.Predicate;
 import org.hypertrace.core.datamodel.AttributeValue;
 import org.hypertrace.core.datamodel.RawSpan;
 import org.hypertrace.core.spannormalizer.TraceIdentity;
 
 public class ByPassPredicate implements Predicate<TraceIdentity, RawSpan> {
+  private static final String ALL = "*";
   private static final String SPAN_BYPASSED_CONFIG = "processor.bypass.key";
-  private String bypassKey;
+  private static final String SPAN_BYPASSED_OVERRIDE_CONFIG = "processor.bypass.override.tenants";
```

Comment:
do we want to bypass all spans for this tenant or the spans of this tenant with specific agent types?